sieve_of_eratosthenes/sieve_of_eratosthenes.py

```diff
@@ -7,20 +7,20 @@ def sieve_of_eratosthenes(n):
     :param n: Number upto which to calculate primes
     :return: A boolean list contaning only primes
     """
-    primes = [True] * (n+1)  # Setting all as True initially
+    primes = [True] * (n + 1)  # Setting all as True initially
     primes[0] = primes[1] = False  # Since 0 and 1 are not primes
-    sqrt_n = math.ceil(math.sqrt(n))
+    sqrt_n = int(math.ceil(math.sqrt(n)))
```

Comment:
@Monal5031: Always make sure that you check for version compatibility of Python code.